README.md

```diff
@@ -229,6 +229,23 @@ To increase or decrease the logging level for the S3 plugin, add an `s3_resource
 },
 ```
 
+## Troubleshooting
+
+### Handling Reduce Your Request Rate Errors
```

Comment:
Wrap `Reduce Your Request Rate` in double quotes so that phrase appears as a single unit.

Reply:
done